scalastyle-config.xml

```diff
@@ -39,11 +39,11 @@
             <parameter name="regex"><![CDATA[[A-Z][A-Za-z]*]]></parameter>
         </parameters>
     </check>
-    <check level="warning" class="org.scalastyle.scalariform.ObjectNamesChecker" enabled="true">
```

Comment:
Why is this commented out and not deleted if not being used?

Reply:
Will delete it.